tdrs-backend/tdpservice/search_indexes/models/ssp.py

```diff
@@ -1 +1,180 @@
 """Models representing parsed SSP data file records submitted to TDP."""
+
+from django.db import models
+from django.contrib.contenttypes.fields import GenericRelation
+from tdpservice.parsers.models import ParserError
+
+
+class SSP_M1(models.Model):
+    """
+    Parsed record representing an SSP M1 data submission.
+
+    Mapped to an elastic search index.
+    """
+
+    error = GenericRelation(ParserError)
+    RecordType = models.CharField(max_length=156, null=True, blank=False)
+    RPT_MONTH_YEAR = models.IntegerField(null=True, blank=False)
```

Comment:
should these all be `null=True, blank=False`

Reply:
done because some "blank" values couldn't be handled elegantly and would pass validation but fail parsing. could add field defaults?

Reply:
For these, I would prefer null over blank/default values so I believe you have made the right choice.